src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  width: calc($cell-size * 7 + $gap * 6);
+  padding: 10px;
+
+  &__day {
```

Comment:
The use of `&__day` suggests a BEM element, but it should be a direct descendant of a block. Ensure that the BEM methodology is correctly applied.